fireapp/lib/presentation/constraint_form/text_input_field.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:fireapp/presentation/constraint_form/base_input_field.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/src/widgets/framework.dart';
+
+class SchedulerTextInputField extends BaseInputField {
```

Comment:
Just name this TextInputField (unless that name is already taken, then call it FireAppTextInputField). You've written a nice, generalised class here that can be used across the whole app, don't sell it short!